pkg/alert/relabel_controller.go

```diff
@@ -61,6 +61,8 @@ type RelabelConfigController struct {
 }
 
 // NewRelabelConfigController returns a new RelabelConfigController instance.
+//
+//nolint:errcheck
```

Comment:
I'd prefer `nolint:errcheck` to be inserted where we explicitly agree to skip err checking.